src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -190,6 +224,19 @@ private void updateCaseInCcd(
                 existingCase.getId(),
                 exceptionRecord.id
             );
+
+            return Either.right(true);
+        } catch (FeignException.UnprocessableEntity exception) {
+            String msg = format("Service returned 422 Unprocessable Entity response when trying to update "
+                + "case for %s jurisdiction with case Id %s based on exception record with Id %s. "
+                + "Service response: %s",
+                exceptionRecord.poBoxJurisdiction,
+                existingCase.getId(),
+                exceptionRecord.id,
+                exception.contentUTF8());
+            log.error(msg, exception);
+
+            return Either.left(singletonList(msg));
```

Comment:
Sorry, this bit is my fault - I thought I was making a comment on catching exceptions resulting from update API's response, not CCD's response. This bit should be removed.